src/Voron/Debugging/StorageReportGenerator.cs

```diff
@@ -161,7 +162,8 @@ private List<TempBufferReport> GenerateTempBuffersReport(VoronPathSetting tempPa
                     return new TempBufferReport
                     {
                         Name = file.Name,
-                        AllocatedSpaceInBytes = file.Length
+                        AllocatedSpaceInBytes = file.Length,
+                        Type = TempBufferType.RecyclableJournal
```

Comment:
Do we want to display this in Storage Report in Studio differently maybe?

Reply:
We could mark it's in `Journals` dir. I'll reopen the ticket for studio adjustments